astropy/coordinates/baseframe.py

```diff
@@ -637,14 +641,16 @@ def _infer_repr_info(cls, repr_info):
         return repr_info
 
     @classmethod
-    def _create_readonly_property(cls, attr_name, value):
+    def _create_readonly_property(cls, attr_name, value, docstring=None):
```

Comment:
This is a nitpick, especially because ``_create_readonly_property`` is private, but python's https://docs.python.org/3.8/library/functions.html#property uses ``doc`` over ``docstring``. Just for consistency.

Reply:
Done